.github/workflows/build.yml

```diff
@@ -6,8 +6,6 @@ on:
   pull_request:
   push:
     branches:
-      - main
```

Comment:
We don't need to run `v0.37.x` workflows on the `main` branch - the `main` branch has its own workflows.